marlowe-website/src/index.njk

```diff
@@ -15,245 +15,414 @@
+            <div class="relative md:w-5/6 w-11/12 md:mt-14 mt-8">
+              <!--Row-->
+              <div class="relative w-full">
+                <!--Column-->
+                <a href="https://staging.marlowe-dash.iohkdev.io" target="_blank" class="primary-button text-lg">Try it</a>
+                <img loading="lazy" class="w-32 absolute left-52 bottom-2 xl:block hidden" src="{% static './static/img/arrow-light.svg' %}" alt="">
+              </div>
+              <!--Column-->
+            </div>
+          </div>
+          <!--Left column-->
+
+          <div class="lg:row-start-auto row-start-1 w-11/12 mx-auto mt-4 mb-8 md:py-16 py-10 px-5 bg-white shadow-lg h-auto rounded-lg">
+            <img loading="lazy" class="lg:w-11/12 md:w-3/4 w-11/12 lg:h-80 md:h-80 h-60 object-cover mx-auto" src="{% static './static/img/blockly.gif' %}" alt="">
+          </div>
+
+          <!--<div class="lg:row-start-auto row-start-1"> <img loading="lazy" class="w-full" src="{% static './static/img/marlowe-blockly-blocks.png' %}" alt=""> </div>-->
         </div>
-        <div class="py-6">
-          <h4 class="font-bold">Safe and Easy to Use</h4>
-          <p>Secure by design. Built for anyone to easily understand and safely use.</p>
+        <!--Row-->
+
+      </div>
+
+      <div class="mx-3 md:mx-5vw flex justify-center">
+        {# Defines the margins #}
+
+        <div class="relative w-full max-w-screen-widest text-center md:pb-10 pb-0 z-10">
+          <!--Row-->
+
+          <div class="">
+            <!--Column-->
+            <h3 class="pt-16 lg:text-xl text-lg font-semibold pb-4 md:block text-green">See how it works</h3>
+            <div class="relative xl:w-3/4 w-full mx-auto">
+              <!--Video thumbnail-->
+              <div class="absolute md:bottom-5 md:right-5 bottom-3 right-3">
+                <a href="#" class="scale"><img loading="lazy" class="w-full" src="{% static './static/img/marlowe-play-btn.svg' %}" alt=""></a>
+              </div>
+              <img loading="lazy" class="w-full mx-auto" src="{% static './static/img/marlowe-build-video-thumbnail.png' %}" alt="">
+            </div>
+            <!--Video thumbnail-->
+          </div>
+          <!--Column-->
+
         </div>
+
+        <!--Row-->
+
       </div>
-      <div class="md:row-start-1 md:row-end-3">
-        <div class="bg-white rounded px-2 pb-2 shadow-lg">
-          <div class="ml-3 py-3 flex">
-            <div class="bg-mac-red w-3 h-3 rounded-full mr-2"></div>
-            <div class="bg-mac-yellow w-3 h-3 rounded-full mr-2"></div>
-            <div class="bg-mac-green w-3 h-3 rounded-full"></div>
+
+      <div class="mx-3 md:mx-5vw flex justify-center">
+        {# Defines the margins #}
+
+        <div class="relative grid grid-cols-1 lg:grid-cols-2 w-full max-w-screen-widest  mt-8 z-10">
+          <!--Row-->
+
+          <div class="w-11/12 mx-auto my-10 shadow-lg h-auto rounded-lg" style="background-color:#232d3d;">
```

Comment:
if that colour is not already present, you should add it to our colour pallete

https://tailwindcss.com/docs/customizing-colors

and then use `bg-custom-colour`

Reply:
also `w-11/12` for margins and `mx-auto` for centering should be changed for `mx-{val}`for margin and `flex justify-center` for centering.